wkcuber/metadata.py

```diff
@@ -122,11 +134,16 @@ def create_parser():
         help="Scale of the dataset (e.g. 11.2,11.2,25)",
         default="1,1,1")
 
+    parser.add_argument(
```

Comment:
I would invert the logic here: Explicitly flag to search for max id, because it is such a resource-intense activity